### ISSUE TITLE
Razer Mouse side buttons not recognized

### ISSUE THREAD

**Issue by mgiu96411** [pqrs-org/Karabiner-Elements#3081]
Hey all!
I have a Razer Basilisk V2, which has extra buttons on the left side. They used to be recognized as mouse button 4 and 5, but they are no longer recognized at all. I open event-viewer and every other button on the mouse is recognized (even clicking the scroll-wheel - button 3), except for these two. I have reinstalled it fresh a few times already, with computer restarts in between. I have an M1 Pro Macbook Pro. When I first got it, this worked.

**Comment by Ryanauger95** [comment]
Any update on this?

**Comment by mgiu96411** [comment]
Not on my end. Kinda just gave up.

**Comment by keni4music** [comment]
I have a Razer Naga Pro and spoke to Razer support about support for Mac. They told me that I would have limited or no support with the extra buttons if attempting to run wireless. Via wire, everything works. Is this the same for you?

**Comment by mgiu96411** [comment]
Mine is only wired

**Comment by raphaelluchini** [comment]
Any update on that or clue why is not working? Would be nice to have that back :/

**Comment by lebel** [comment]
Also, the buttons on the mouse wheel (ie. left and right pull clicks)

**Comment by asnaek** [comment]
I figured out why mine wasn't working:

- The last thing I had been trying to do in Synapse before uninstalling it, was assigning macros to button 4 & button 5.
- I reinstalled Synapse and reset the mouse/profile to the defaults.
- I noticed that EventViewer was detecting mouse4/mouse5 again, but only while Synapse was open.
- From Synapse, I remapped mouse button 4 to mouse button 4 & mouse button 5 to mouse button 5 (sounds stupid, but this is what worked).
- I quit Synapse I noticed that EventViewer was detecting the buttons all the time.
- I uninstalled Synapse & rebooted, EventViewer still detects the buttons.

TL;DR: If you had macros assigned to buttons, or a completely default profile, in both cases that seems to prevent those buttons from working at all when Synapse isn't running - remap the buttons to non-Synapse-required ones then can get rid of it.
